### PR TITLE
Init Electra (EIP6110 + EIP7002)

### PR DESCRIPTION
**Pull request by hwwhww**
Merged EIP6110 and EIP7002 specs into new Electra specs.

/cc @djrtwo

### CODE REVIEW

File: specs/electra/beacon-chain.md [hunk 1]
```diff
@@ -30,35 +34,37 @@
...
+| `MAX_DEPOSIT_RECEIPTS_PER_PAYLOAD` | `uint64(2**13)` (= 8,192) | *[New in Electra:EIP6110]* Maximum number of deposit receipts allowed in each payload |
+| `MAX_EXECUTION_LAYER_EXITS` | `2**4` (= 16) |  *[New in Electra:EIP7002]* |
 
 ## Containers
 
 ### New containers
 
 #### `DepositReceipt`
 
+*Note*: The container is new in EIP6110.
```
Comment (dapplion) [inline specs/electra/beacon-chain.md:66]: This note can be inferred from 
```
 deposit_receipts: List[DepositReceipt, MAX_DEPOSIT_RECEIPTS_PER_PAYLOAD]  # [New in Electra:EIP6110]`
```
not sure if it's worth the percent and now we may have to annotate all containers

File: specs/electra/beacon-chain.md [hunk 2]
```diff
@@ -179,20 +197,20 @@ class BeaconState(Container):
 def process_block(state: BeaconState, block: BeaconBlock) -> None:
     process_block_header(state, block)
     process_withdrawals(state, block.body.execution_payload)
-    process_execution_payload(state, block.body, EXECUTION_ENGINE)  # [Modified in EIP6110]
+    process_execution_payload(state, block.body, EXECUTION_ENGINE)  # [Modified in Electra:EIP6110]
     process_randao(state, block.body)
     process_eth1_data(state, block.body)
-    process_operations(state, block.body)  # [Modified in EIP6110]
+    process_operations(state, block.body)  # [Modified in Electra:EIP6110]
```
Comment (mkalinin) [inline specs/electra/beacon-chain.md:203]:
```suggestion
    process_operations(state, block.body)  # [Modified in Electra:EIP6110:EIP7002]
```

**Review by mkalinin — COMMENTED** [review]
Looks good to me except for a couple of nits!

`fork.md` still refers to EIP-6110: "This document describes the process of EIP-6110 upgrade.” (couldn’t make a proper link to this line)

**Review by djrtwo — APPROVED** [review]
nice work!

**Comment by StefanBratanov** [comment]
I raised a small PR https://github.com/ethereum/consensus-specs/pull/3629 for an `EIP-6110` function rename. I am not sure if a comment here would have been more appropriate.

**Review by lucassaldanha — APPROVED** [review]
Well done!

**Pull request merged**